src/main/java/io/cdap/plugin/google/common/GoogleAuthBaseConfig.java

```diff
@@ -177,6 +192,9 @@ private boolean validateAuthType(FailureCollector collector) {
   }
 
   private boolean validateOAuth2Properties(FailureCollector collector) {
+    if (containsMacro(CLIENT_ID) || containsMacro(CLIENT_SECRET) || containsMacro(REFRESH_TOKEN)) {
+      return false;
+    }
```

Comment:
This check looks redundant, `containsMacro` is already being checked in `checkPropertyIsSet` below.

Can we remove this ?

Reply:
removed.